src/Data/Number.hs

```diff
@@ -0,0 +1,24 @@
+module Data.Number (
+  inexactNumber
```

Comment:
I seems a little odd to export constructors and functions which do basically the same